packages/framework/block-std/src/range/range-binding.ts

```diff
@@ -245,20 +249,21 @@ export class RangeBinding {
     if (text === this._prevTextSelection) {
       return;
     }
-
     // wait for lit updated
     this.host.updateComplete
       .then(() => {
         const model = text && this.host.doc.getBlockById(text.blockId);
         const path = model && this.host.view.calculatePath(model);
 
-        const eq =
-          text && this._prevTextSelection && path
-            ? text.equals(this._prevTextSelection.selection) &&
-              path.join('') === this._prevTextSelection.path.join('')
-            : false;
+        if (this.host.event.active) {
+          const eq =
+            text && this._prevTextSelection && path
+              ? text.equals(this._prevTextSelection.selection) &&
+                path.join('') === this._prevTextSelection.path.join('')
```

Comment:
please also remove path here, use block id instead

Reply:
The pr has been automatically merged by graphite, I will open another pr to remove path.